packages/groq/src/define.ts

```diff
@@ -0,0 +1,16 @@
+/**
```

Comment:
As explained in the PR description, we cannot add `defineQuery` to the top-level `groq` package since that might break CommonJS users of the package.